File: semgrep-core/tests/patterns/ts/metavar_func_def.ts
// ERROR:
function foo() {
    foo(1,2);
}

// ERROR:
function bar(bar1:number,bar2:number,bar2:number) {
    foo(1,2);
    bar(1,2,3);
}

// ERROR:
function foobar(bar1: number): number {
    foo();
}

File: semgrep-core/tests/patterns/go/metavar_func_def.go
package Foo

// ERROR:
func foo() {
    foo (1,2)
}

// ERROR:
func bar(bar1, bar2, bar3) {
    return 1
}

// ERROR:
func foobar(bar1) int {
    bar(1, 2, 3)
    foo()
    return 2
}
